training/tf/tfprocess.py

```diff
@@ -382,14 +381,14 @@ def construct_net(self, planes):
         # Value head
         conv_val = self.conv_block(flow, filter_size=1,
                                    input_channels=RESIDUAL_FILTERS,
-                                   output_channels=1)
-        h_conv_val_flat = tf.reshape(conv_val, [-1, 8*8])
-        W_fc2 = weight_variable([8 * 8, 256])
-        b_fc2 = bias_variable([256])
+                                   output_channels=32)
+        h_conv_val_flat = tf.reshape(conv_val, [-1, 32*8*8])
+        W_fc2 = weight_variable([32 * 8 * 8, 128])
+        b_fc2 = bias_variable([128])
```

Comment:
Why down to 128 here?  And it seems the C++ side is still at 256?

Reply:
Oh thanks! That's a bug on the C++ side. I went down to 128 to see if it matters, this is what the 64x6 network uses currently.

Reply:
Are you referring to this line? https://github.com/glinscott/leela-chess/blob/master/src/Network.cpp#L664 

What do you think we should do. Set to 128? Or retrain the current net or wait for the final results before merging/not merging?

Reply:
Yup, exactly.  And we should fix the 256 to be NUM_VALUE_CHANNELS as well.  I missed a few places when converting the old hardcoded numbers over :).  Maybe easiest to leave it at 256?  As that's what AlphaGo Zero was using at least.

I'm fine with changing it to 128 as well though, it might need to be something we experiment with.

Reply:
Ok, I shall cleanup the code a bit in this PR such that it atleast corresponds to the tensorflow model.  I'll use some static constants such that it's easy to change. After that merging will be up to you ;-) Maybe wait until more results are in. Ideally I should train another network with 256 being the only change. It however takes about 72 hrs though...

Reply:
Just verified: the PV corresponding to Stockfish's PV is still among the solutions. It just surfaces a bit faster as the network is a bit faster.